src/Invoker.ts

```diff
@@ -0,0 +1,114 @@
+import * as http from "http";
+import * as https from "https";
+import * as path from "path";
+import * as URL from "url";
+
+export class Invoker {
+    public static async invokeExpressFile(handler: string, port: number, jsonRequest: any, restOfPath?:string): Promise<any> {
+        const handlerParts = handler.split(".");
+        const fileName = handlerParts.join("/") + ".js";
+        const fullPath = path.join(process.cwd(), fileName);
+
+        // Ensure the cache is removed always to be able to start the server on command
+        delete require.cache[require.resolve(fullPath)];
+        const handlerModule = require(fullPath);
+
+        if (!handlerModule.close) {
+            throw new Error("The web server needs to be in the module.exports")
+        }
+
+        await new Promise(resolve => {
+            handlerModule.on("listening", (server: any) => {
+                resolve();
+            });
+        });
+
+        const urlString = "http://127.0.0.1:" + port;
+
+        const response = await Invoker.invokeWithURLString(urlString, jsonRequest);
+
+        //After we process the request, we close the server before sending the next one.
+        await new Promise(resolve => handlerModule.close(resolve));
+
+        return response;
+    }
+
+    public static invokeHandler(handler: string, jsonRequest: any): Promise<any> {
+        const handlerParts = handler.split(".");
+        const functionName = handlerParts[handlerParts.length - 1];
```

Comment:
Take a look at the module handler in virtual-alexa - we should do the same processing:
https://github.com/bespoken/virtual-alexa/blob/master/src/impl/ModuleInvoker.ts#L5

In particular, we do not want to use the split on "." - which causes problems. The code also defaults the function name to handler if no function name is provided